src/config/dynamic-theme-fixes.config

```diff
@@ -99,6 +99,15 @@ INVERT
 
 ================================
 
+*.crm*.dynamics.com
+
+CSS
+.ms-Label {
+    color: var(--dark-neutral-text);
```

Comment:
Maybe it should look like
```
color: var(--darkreader-neutral-text) !important;
```

Reply:
Made an Approve by accident

Reply:
I had no problem without `!important`. But, if it's better to use `!important`, I can make that change.

Reply:
Please update the variable and the !important, check if it works and I will fix the conflict (probably similar lines were edited in another Pull Request by someone else).